.travis.yml

```diff
@@ -2,7 +2,7 @@ matrix:
   include:
     - language: java
       script:
-        - make validate-canonical-match
+        - 'if [[ "$TRAVIS_PULL_REQUEST" = "false" && "master" = "$TRAVIS_BRANCH" ]]; then make deploy-license-data; else make validate-canonical-match; fi'
     - language: node_js
       node_js:
-      - "9"
+      - "9"
```

Comment:
Can we keep the trailing newline?  It makes it easier to read the file when `cat`ing it to the terminal, because the next prompt will start on a new line.